lib/public/js/docs-nav.js

```diff
@@ -6,8 +6,8 @@ const Navbar = require('./vendor/navbar');
 
 const internals = {};
 
-const FIXED_HEADER_HEIGHT = 80;
-const TOP_PADDING = 35;
+const FIXED_HEADER_HEIGHT = 60;
```

Comment:
Should we change the code to not use these on mobile, since the nav is no longer fixed?

Reply:
Probably, it doesn't cause huge issues as far as I can see. We should definitely rethink it if it is!